symplyphysics/laws/electricity/electromotive_force_induced_in_sinusoidally_rotating_coil.py

```diff
@@ -22,16 +22,19 @@
 #. The rotation of the coil is uniform.
```

Comment:
I wonder why law name is about sinusoidal rotation. Sinusoidal rotation is not uniform and proof states that it is rotating with constant angular speed.

Reply:
I reworded it so that it's about the coil's oscillation rather than rotation.